File: api-user/src/main/java/fr/formation/model/Utilisateur.java
package fr.formation.model;

import java.time.LocalDate;

import org.hibernate.annotations.UuidGenerator;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.Setter;

@Entity
@Table(name = "[user]")
@Getter @Setter
public class Utilisateur {
    @Id
    @UuidGenerator
    @Column(name = "usr_id")
    private String id;
    
    @Column(name = "usr_username")
    private String username;

    @Column(name = "usr_password")
    private String password;

    @Column(name = "usr_email")
    private String email;

    @Column(name = "usr_naissance")
    private LocalDate dateNaissance;
}

File: python-flask/api-user/model/Utilisateur.py
from flask_sqlalchemy import SQLAlchemy
from sqlalchemy.dialects.postgresql import UUID
import uuid

db = SQLAlchemy()

class Utilisateur(db.Model):
    __tablename__ = 'user'

    id = db.Column('usr_id', UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
    username = db.Column('usr_username', db.String)
    password = db.Column('usr_password', db.String)
    email = db.Column('usr_email', db.String)
    dateNaissance = db.Column('usr_date_naissance', db.Date)
    